app/models/micropost.rb

```diff
@@ -0,0 +1,23 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+
+  validates :user_id,
```

Comment:
t nghĩ cái validates :user chặt chẽ hơn nhưng sẽ mất công tìm trong database, dùng cái validates :user_id là đủ rồi vì khó xảy ra trường hợp cần đến check user (action create microposts tạo từ current_user và đã check logged in user)